relay-server/src/endpoints/minidump.rs

```diff
@@ -185,4 +192,110 @@ mod tests {
         let garbage = b"xxxxxx";
         assert!(validate_minidump(garbage).is_err());
     }
+
+    #[tokio::test]
+    async fn test_minidump_multipart_attachments() -> anyhow::Result<()> {
+        let _body_data2 : &[u8] =
```

Comment:
nit: I think `_` isn't required:
```suggestion
        let body_data2 : &[u8] =
```

Reply:
Sorry, that was a leftover from testing with various multipart bodies.